src/crypto/crypto.ts

```diff
@@ -52,12 +53,26 @@ const keychainPromises = {
    * @param account The keychain account name.
    */
   getPassword(_keychain: KeyChain, service: string, account: string): Promise<CredType> {
```

Comment:
"password"...would keyChainEntry or keychainItem be a better description of what we're actually getting here?  This is getting back the key we use for decrypting stuff?